extract.sh

```diff
@@ -32,7 +33,8 @@ UPSTREAM_FILES="sys/bpf.txt sys/dri.txt sys/fuse.txt sys/input.txt sys/ipc.txt
 
 ANDROID_FILES="sys/tlk_device.txt"
 
-if [ "$BUILD_FOR_ANDROID" == "no" ]; then
+if [ "$BUILD_FOR_ANDROID" == "no" ]; 
+then
```

Comment:
This new formatting is not consistent with the rest of the file, so no.
The other change is fine.